04/app.js

```diff
@@ -0,0 +1,46 @@
+const arr = getArray(10,200);
+const largest = getLargest(arr);
+
+const avg = getAvg(largest);
+
+console.log('Średnia to' , avg)
+
+
+function getAvg(x) {
+
+   const y = x;
```

Comment:
Ponieważ `y` nie jest zmienane wew. tej funkcji to można po prostu wszędzie wykorzystywać `x` zamiast tworzyć dodatkową zmienną.